docs/gettingstarted/structuring.md

```diff
@@ -25,6 +25,86 @@ What is `@Mod`?
 
 This is an annotation indicating to the Forge Mod Loader that the class is a Mod entry point. It contains various metadata about the mod. It also designates the class that will receive `@EventHandler` events. More information can be found at... (Coming Soon)
 
+All components of @Mod() are written inside the parantheses and chained together. There's typically only one @Mod in your mod as a whole, sitting on top of your entry class.
+
+Components:
+
+`modid` [Required]
+This identifies your mod uniquely to Forge and should probably be the first thing you enter. It's only used internally, but can be seen by users who dig around. Other mods also require this ID if they want to find each other (APIs for example).
```

Comment:
"and should probably be the first thing you enter. It's only used internally, but can be seen by users who dig around"

Too much fluff.

Something like
"Uniquely identifies your mod to FML and all other mods. Should not be changed after you release your mod, as FML would then consider it a new, separate mod"